mods/cnc/mod.yaml

```diff
@@ -154,6 +154,7 @@ ContentInstaller:
 		.: conquer.mix, desert.mix, general.mix, scores.mix, sounds.mix, temperat.mix, winter.mix
 	ShippedSoundtracks: 2
 	MusicPackageMirrorList: http://www.openra.net/packages/cnc-music-mirrors.txt
+	TFDIndexes: 66, 45, 42, 65, 68, 67, 71, 47, 49, 60, 75, 73
```

Comment:
the numbers correspond to where the file headers references appear in data1.hdr
It's to get around the fact that ISCF will allow for 2 files of identical name in the same directory, maybe ISCFFileIndexes or ISCFFileHeaderIDs?

The files on the TFD disk seem different to the original CDs for RA and TD for example [TFD is missing an allies.mix and a russian.mix](https://gist.github.com/DSUK/e243103217bac918b175#file-tfd-files-L943)
[Tiberian sun is also missing almost all the files mentioned in the mod.yaml](https://gist.github.com/DSUK/e243103217bac918b175#file-tfd-files-L338)

I suspect they might have use *.mix files from one from one of the add-ons or otherwise coalesced the mix files in some way, Not 100% sure how i would test for that though.